src/main/java/org/jenkinsci/plugins/DependencyTrack/DependencyTrackPublisher.java

```diff
@@ -223,7 +223,7 @@ public void perform(@Nonnull final Run<?, ?> build,
                 final FindingParser parser = new FindingParser(build.getNumber(), jsonResponseBody).parse();
                 final ArrayList<Finding> findings = parser.getFindings();
                 final SeverityDistribution severityDistribution = parser.getSeverityDistribution();
-                final ResultAction projectAction = new ResultAction(findings, severityDistribution);
+                final ResultAction projectAction = new ResultAction(build, findings, severityDistribution);
```

Comment:
Pass the current build into the BuildAction so that it can tie the ProjectActions back to this build.